packages/cli/commons/src/preconditions/isWebAuthenticated.ts

```diff
@@ -0,0 +1,17 @@
+import {Config} from '../config/config';
+import globalConfig from '../config/globalConfig';
+import {PreconditionError} from '../errors/preconditionError';
+
+const PRECONDITION_ERROR_CATEGORY = 'Wrong Authentication Type';
+
+export function IsWebAuthenticated() {
```

Comment:
What about adding parameters to the `IsAuthenticated()` precondition ?

Say, an array of supported authentication scheme.

By default it could be an empty array (or an array with all schemes).

This just seems weird to me: 

```
@Preconditions(
    IsAuthenticated(),
    IsWebAuthenticated(),
    ....
)
```

vs: 
```
@Preconditions(
    IsAuthenticated(['OAuth']),
    ....
)
```

Reply:
Mh yeah we could do that too

Reply:
Done ✔️